infra/tf/README.md

```diff
@@ -10,7 +10,7 @@ Terraform is our tool of choice for automating our 'cloud infrastructure'. In pa
 * [Going public](#going-public)
 * [Publishing a release](#publishing-a-release)
 * [Updating a module for a new version of Terraform](#updating-a-module-for-a-new-version-of-Terraform)
```

Comment:
```suggestion
* [Updating a module for a new version of Terraform](#updating-a-module-for-a-new-version-of-terraform)
```
Since you're making a change for consistency this anyway 😂